src/util/config/config.cpp

```diff
@@ -597,6 +597,10 @@ namespace dxvk {
     { R"(\\PortRoyale3\.exe$)", {{
       { "d3d9.allowDoNotWait",           "False" },
     }} },
+    /* DayZ */
+    { R"(\\DayZ_x64\.exe$)", {{
+      { "d3d11.cachedDynamicResources",     "cr" },
+    }} },
```

Comment:
We generally have D3D11 and D3D9 options separated so this should probably go into the D3D11 section way above this.

Reply:
I was looking if there is some ordering or anything and completely missed the DX9 banner, my bad